cmd/appliance/upgrade/prepare.go

```diff
@@ -140,13 +145,6 @@ func NewPrepareUpgradeCmd(f *factory.Factory) *cobra.Command {
 			return errs
 		},
 		RunE: func(c *cobra.Command, args []string) error {
-			h, err := opts.Config.GetHost()
-			if err != nil {
-				return fmt.Errorf("could not determine hostname for %s", err)
-			}
-			if err := network.ValidateHostnameUniqueness(h); err != nil {
-				return err
-			}
```

Comment:
same as previous comment, I don't understand how removing this is related to actual hostname changes. This error message is show if the config value for controller API resolves to more then 1 ip, it should not be related to this problem

Reply:
Same answer as above as well

Reply:
oh wait, now I see what you mean, but why are we validating the config host? No-one has asked for that as far as I know.